packages/ui-extensions/src/docs/shared/components/NumberField.ts

```diff
@@ -3,7 +3,7 @@ import {SharedReferenceEntityTemplateSchema} from '../docs-type';
 const data: SharedReferenceEntityTemplateSchema = {
   name: 'NumberField',
   description:
-    'Use `s-number-field` when you need to collect numerical input from merchants. It provides appropriate keyboard settings and validation for numerical values.',
+    'Collect numerical input from merchants with optimized keyboard settings and built-in validation.',
```

Comment:
I think "values" is consistent with the email / number fields.

```suggestion
    'Collect numerical values from users with optimized keyboard settings and built-in validation.',
```